ToyBox/classes/MainUI/BagOfTricks.cs

```diff
@@ -274,6 +274,15 @@ public static class BagOfTricks {
                 () => UI.Toggle("Allow Item Use From Inventory During Combat", ref settings.toggleUseItemsDuringCombat),
                 () => UI.Toggle("Ignore Alignment Requirements for Abilities", ref settings.toggleIgnoreAbilityAlignmentRestriction),
                 () => UI.Toggle("Remove Level 20 Caster Level Cap", ref settings.toggleUncappedCasterLevel),
+                () => {
+                    using (UI.HorizontalScope()) {
```

Comment:
This is redundant as the HSStack wraps the lambda in one.  I would remove it (which I will do after I merge this time)